src/vivarium/framework/results/stratification.py

```diff
@@ -45,9 +47,9 @@ def __post_init__(self):
                 )
             self.mapper = self._default_mapper
             self.is_vectorized = True
-        if not len(self.categories):
+        if len(self.categories) == 0:
```

Comment:
nit: I think PEP 8 recommends `if not self.categories:` here, but obviously this is fine as well.

Reply:
Interesting, pylint was actually saying _not_ to do that - though I didn't track down the source

Reply:
Ohhh, I see, it doesn't like `if not len(seq)` since you can just do `if not seq` instead. Not sure why it's not yelling at me about the `== 0` though...

```
For sequences, (strings, lists, tuples), use the fact that empty sequences are false:
# Correct:
if not seq:
if seq:
# Wrong:
if len(seq):
if not len(seq):
```